src/battle_anim_rock.c

```diff
@@ -456,7 +478,14 @@ void AnimRockFragment(struct Sprite *sprite)
 // Swirls particle in vortex. Used for moves like Fire Spin or Sand Tomb
 void AnimParticleInVortex(struct Sprite *sprite)
 {
-    InitSpritePosToAnimBattler(gBattleAnimArgs[6], sprite, FALSE);
+    if (IsDoubleBattle() //got a little lazy here will fix later
+    && (gAnimMoveIndex == MOVE_BLEAKWIND_STORM 
```

Comment:
bleakwind storm etc should be using `createvisualtaskontargets` to create the gust on each target, or `SetAverageBattlerPositions` to create the gust centered on the targets